stetho/src/main/java/com/facebook/stetho/inspector/elements/android/FragmentDescriptor.java

```diff
@@ -0,0 +1,86 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+package com.facebook.stetho.inspector.elements.android;
+
+import android.view.View;
+
+import com.facebook.stetho.common.LogUtil;
+import com.facebook.stetho.common.android.FragmentAccessor;
+import com.facebook.stetho.common.android.FragmentApi;
+import com.facebook.stetho.inspector.elements.AttributeAccumulator;
+import com.facebook.stetho.inspector.elements.ChainedDescriptor;
+import com.facebook.stetho.inspector.elements.DescriptorMap;
+
+final class FragmentDescriptor
+    extends ChainedDescriptor<Object> implements HighlightableDescriptor {
+  private static final String ID_ATTRIBUTE_NAME = "id";
+  private static final String TAG_ATTRIBUTE_NAME = "tag";
+
+  private static final FragmentDescriptor sInstance;
+  static {
+    sInstance = new FragmentDescriptor();
+  }
+
+  public static DescriptorMap register(DescriptorMap map) {
+    Class<?> supportFragmentClass = FragmentApi.tryGetSupportFragmentClass();
+    if (supportFragmentClass != null) {
+      LogUtil.i("Registering support Fragment descriptor");
```

Comment:
This seems to me like it should be `.d`.  `.i` will log in the caller application.